packages/@okta/vuepress-site/docs/guides/quickstart/register-app/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Register your app
+---
+
+In this section, we hook up your Okta org to the app that's going to use it for sign-in. To tell Okta about the app, and to collect the pieces of information you need to integrate into your app, you use the Developer Console's app screen.
+
+For simplicity, in this run-through let's assume that your app is a web app, i.e., a server-side app, and that you want to use the Okta-hosted sign-in page, rather than implement or host your own sign-in page.
+
+To start, find the example applications, provided by Okta on GitHub, for the language or framework you work in:
+
+| Language / Framework | Link                                               |
+|----------------------|----------------------------------------------------|
+| ASP.Net              | <https://github.com/okta/samples-aspnet>           |
+| ASP.NET Core         | <https://github.com/okta/samples-aspnetcore>       |
+| ASP.Net Web Forms    | <https://github.com/okta/samples-aspnet-webforms>  |
+| Java Servlet         | <https://github.com/okta/samples-java-servlet>     |
+| Java Spring          | <https://github.com/okta/samples-java-spring>      |
+| JavaScript Angular   | <https://github.com/okta/samples-js-angular>       |
+| JavaScript React     | <https://github.com/okta/samples-js-react>         |
+| JavaScript Vue       | <https://github.com/okta/samples-js-vue>           |
+| Golang               | <https://github.com/okta/samples-golang>           |
+| Node.js Express      | <https://github.com/okta/samples-nodejs-express-4> |
+| Python Flask         | <https://github.com/okta/samples-python-flask>     |
+| PHP                  | <https://github.com/okta/samples-php>              |
+
+Specifically, look for the example app provided for that language/framework for implementing authentication inr a web app using the Okta-hosted login page (here are samples provided for many other purposes).
+
+After getting the sample code, you need to hook it up to your Okta org by registering the app in Developer Console and then getting some values from Okta, which you need to set in the app.
+
+### Tell Okta about the app
+
+You need to register the app in Developer Console:
```

Comment:
```suggestion
Using the Okta CLI, run `okta apps create`.

1. Give your app a name.
2. Select **Web** for Type of Application.
3. Select **Other**.
4. Use the default Login redirect URI, `http://localhost:8080/callback`

You can also use the Developer Console:
```

Reply:
@bdemers We might want to make the default Login redirect URI in the CLI the same as the developer console to reduce confusion.

Reply:
@bdemers Even better - change the developer console to use the same URI as the CLI!